### ISSUE TITLE
Evaluation results

### ISSUE THREAD

**Issue by Maplebb** [Con6924/SPM#7]
Thanks for you incredible work. I am confused by the results I get when evaluating your checkpoints.
My problem includes:
 
1. When using the nudity spm, Does "γ = 2.0" means the parameter "facilitate_factor" is set to 2 and applied after calculating the matching score or directly set the scale of the spm to 2 without matching score calculation.
2. When using the spm for artist removal, Does "doubling the semi-permeability γ" means the parameter "facilitate_factor" is set to 2 and applied after calculating the matching score or directly set the scale of the spm to 2 without matching score calculation?
3.  When using the snoopy spm, Is the value of facilitite_factor set to 1?

**Comment by anseryuer** [comment]
Same question here, I really like the demo.ipynb, but the evaluation code is unclear, and cannot directly get the same result in the paper.

**Comment by Con6924** [comment]
Sorry for the unclear description.
1. "γ = 2.0" means the "spm_multiplier" is set to 2;
2. "doubling the semi-permeability γ" means "facilitate_factor" is set to 2;
3. Yes, the snoopy spm uses the default inference settings.

**Comment by JianWang21** [comment]
> Sorry for the unclear description.
> 
> 1. "γ = 2.0" means the "spm_multiplier" is set to 2;
> 2. "doubling the semi-permeability γ" means "facilitate_factor" is set to 2;
> 3. Yes, the snoopy spm uses the default inference settings.

Hi, I retrained the model based on SD-v1.5 using the default settings.

For artist removal, I found that setting facilitate_factor=2 tends to produce better erasure results for the erased artist, but degrades performance for the unerased artists.

Are there any specific settings or hyperparameters that are particularly important or recommended for the artist removal scenario? For example, parameters that differ from those used for nudity or object removal.

Thanks a lot for your work!
